multicluster/controllers/multicluster/commonarea/remote_common_area.go

```diff
@@ -234,7 +234,7 @@ func (r *remoteCommonArea) SendMemberAnnounce() error {
 		}
 		// Add timestamp to force update on MemberClusterAnnounce. Leader cluster requires
 		// periodic updates to detect connectivity. Without this, no-op updates will be ignored.
-		localClusterMemberAnnounce.Annotations[TimestampAnnotationKey] = time.Now().String()
+		localClusterMemberAnnounce.Annotations[TimestampAnnotationKey] = time.Now().Format(time.RFC3339)
```

Comment:
According to the go doc, the time.Now().String() is meant for debugging. So change it to time.RFC3339